### DISCUSSION TITLE
PowerBI early adopters

### DISCUSSION THREAD

**Discussion by buremba** [metriql/metriql#22] [Q&A]
As PowerBI is one of the most advanced BI tools, we need early adopters who are willing to test our PowerBI integration once we have a proof of concept. I will share the relevant links when it's ready, thanks in advance!

You can keep track of the status from #7

**Comment by danielefrigo** [comment]
I'd be happy to test this adapter!

**Reply by jw-aleksandr-zolotukhin** [reply]
sign me up for testing

**Reply by nemccarthy** [reply]
+1

**Reply by boje** [reply]
+1

**Reply by Hakim-Voyager** [reply]
+1

**Reply by lampan1337** [reply]
+1

**Reply by rexpie** [reply]
+1

**Comment by Hakim-Voyager** [comment]
Any updates on the PBI integrations?
